### PR TITLE
Track the model view and projection matrix

### PR DESCRIPTION
**Pull request by mitchej123**

### CODE REVIEW

**Review by embeddedt — APPROVED** [review]
Code looks correct on paper, but I have not actually tried to run it.

File: src/main/java/com/gtnewhorizons/angelica/transform/RedirectorTransformer.java [hunk 1]
```diff
@@ -44,6 +44,8 @@ public class RedirectorTransformer implements IClassTransformer {
     private static final String GL11 = "org/lwjgl/opengl/GL11";
     private static final String GL13 = "org/lwjgl/opengl/GL13";
     private static final String GL14 = "org/lwjgl/opengl/GL14";
+    private static final String PROJECT = "org/lwjgl/util/glu/Project";
```
Comment (Alexdoru) [inline src/main/java/com/gtnewhorizons/angelica/transform/RedirectorTransformer.java:47]: Don't you need to add this in the class constant pool parser ?

Comment (mitchej123) [reply src/main/java/com/gtnewhorizons/angelica/transform/RedirectorTransformer.java:47]: Probably should, however the place it was being used was already identified by one of the other classes.

**Comment by mitchej123** [comment]
I ran it locally in dev, and on one of my old worlds.  The stutters have disappeared.

**Pull request merged**